.env.sample

```diff
@@ -0,0 +1,4 @@
+POSTGRES_HOST=DB_HOST
+POSTGRES_NAME=DB_NAME
```

Comment:
is it correct name? Did you test your application?

Reply:
Yes